### PR TITLE
(SIMP-2946) Change pam_cracklib to pam_pwquality

### PR DESCRIPTION
**Pull request by jeannegreulich**
-changed pam_cracklib.so to pam_pwquality.so
   in pam.d files
  -add /etc/security/pwquality.conf file
  -fixed some puppet strings information

SIMP-3761 #close
SIMP-3762 #close
SIMP-2946 #close

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-pam",
-  "version": "6.0.4",
+  "version": "6.0.5",
```
Comment (jeefberkey) [inline metadata.json]: I think this may be a Y bump

**Comment by trevor-vaughan** [comment]
Just realized that this changes things for both EL6 and EL7. EL6 does not have `pam_pwquality` so we're going to have to support both entries.

**Comment by trevor-vaughan** [comment]
Rebasing

**Comment by jeannegreulich** [comment]
I tested trevor's updates to my fixes on a 6.9 and 7.3 and they worked.

**Pull request merged**